dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/preheat/TrackerPreheat.java

```diff
@@ -267,6 +267,15 @@ public MetadataIdentifier getCategoryOptionComboIdentifier( CategoryCombo catego
     @Getter
     private final Map<String, Relationship> relationships = new HashMap<>();
 
+    /**
+     * Internal map of all relationships in the DB that are duplications of
+     * relationships present in the payload. Two relationship are duplicated if
+     * they have the same relationshipType, and the same relationshipItems, from
+     * and to.
+     */
+    @Getter
+    private final Map<String, Relationship> duplicatedRelationships = new HashMap<>();
```

Comment:
Since English is not my first language take my gut feeling as that 😋 
`duplicateRelationships` feels more correct to me

Reply:
Done